src/main/java/org/geysermc/generator/MappingsGenerator.java

```diff
@@ -479,6 +483,43 @@ public void generateMapColors() {
         }
     }
 
+    public void generateStatistics() {
+        try {
+            // The formatter field is private and does not have an accessor
+            Field formatterField = Stat.class.getDeclaredField("formatter");
```

Comment:
This can also use an access widener - see the access widener file in the resources file and https://www.fabricmc.net/wiki/tutorial:accesswideners for more details. Not a hard requirement though.